src/utils/file.ts

```diff
@@ -0,0 +1,29 @@
+import { exec } from "child_process";
+
+type OSPlatform = "darwin" | "linux" | "win32";
+
+export function getAvailableDiskSpace(path: string = "/"): Promise<number> {
+  return new Promise((resolve, reject) => {
+    const platform = process.platform as OSPlatform;
+    let cmd: string;
+    if (platform === "darwin" || platform === "linux") {
```

Comment:
Recommend dividing this 'if' condition into separate parts for better readability

Reply:
Thank you for the feedback. I have made the necessary changes as per your suggestions!